src/PrestaShopBundle/Routing/LegacyControllerConstants.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Routing;
+
+class LegacyControllerConstants
+{
+    public const CLASS_ATTRIBUTE = '_controller_class';
+    public const INSTANCE_ATTRIBUTE = '_controller_instance';
+    public const ANONYMOUS_ATTRIBUTE = '_controller_anonymous';
+    public const IS_MODULE_ATTRIBUTE = '_controller_is_module';
+    public const IS_ALL_SHOP_CONTEXT = '_controller_is_all_shop_context';
```

Comment:
```suggestion
    public const IS_ALL_SHOP_CONTEXT_ATTRIBUTE = '_controller_is_all_shop_context';
```
Maybe? It's the only one missing the suffix